src/views/DidDowngrade/DidDowngrade.tsx

```diff
@@ -113,6 +114,9 @@ export function DidDowngrade({ identity }: Props): JSX.Element | null {
 
         setStatus('success');
       } catch (error) {
+        if (error instanceof PasswordError) {
+          return;
```

Comment:
Here and in the other files `setSubmitting` is not called, so the state remains `true` and we probably do not want that

Reply:
But `setSubmitting(true)` is not called either, so wouldn't the state remain false?

Reply:
Oh, right!